packages/browser-wallet/src/popup/page-layouts/MainLayout/i18n/en.ts

```diff
@@ -8,6 +8,7 @@ const t = {
         accounts: 'Accounts',
         ids: 'ID cards',
         settings: 'Wallet settings',
+        recovery: 'Restoring your wallet',
```

Comment:
It's a little weird that this settings entry has its own header title, meanwhile the others just show Wallet settings.

Reply:
Yeah, I checked the figma, and it turns out they are all supposed to have their own title. (So I added that)